samples/nrf9160/battery/prj.conf

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Samples default
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# Modem library
+CONFIG_NRF_MODEM_LIB=y
+
+CONFIG_NEWLIB_LIBC=y
+
+# LTE link control
+CONFIG_LTE_LINK_CONTROL=y
+
+# Stacks and heaps
+CONFIG_HEAP_MEM_POOL_SIZE=1024
+CONFIG_MAIN_STACK_SIZE=1536
```

Comment:
Do we need to bump these?

Reply:
I don't understand, what do you mean? Increase size? Or do you mean drop them?

Reply:
I meant is it needed to increase the size from the default values? Did you find that the default were too small?

Reply:
Increased stack size is required or it will bootloop due to stack overflow.